tests/Platforms/SQLite/ComparatorTest.php

```diff
@@ -12,4 +14,25 @@ protected function setUp(): void
     {
         $this->comparator = new Comparator(new SqlitePlatform());
     }
+
+    public function testCompareChangedBinaryColumn(): void
+    {
+        $oldSchema = new Schema();
+
+        $tableFoo = $oldSchema->createTable('foo');
+        $tableFoo->addColumn('id', 'binary');
+
+        $newSchema = new Schema();
+        $table     = $newSchema->createTable('foo');
+        $table->addColumn('id', 'binary', ['length' => 42, 'fixed' => true]);
+
+        $expected             = new SchemaDiff();
+        $expected->fromSchema = $oldSchema;
+
+        self::assertEquals($expected, $this->comparator->compareSchemas(
+            $oldSchema,
+            $newSchema,
+            $this->comparator
+        ));
+    }
```

Comment:
This test fails for SQLite, and probably more platforms, hence why I overrode it, but there is probably a more elegant solution. Let's see how other platforms behave.

Reply:
Actually, it only fails for SQLite: https://github.com/doctrine/dbal/blob/9ddb464b88b6b6869d09123c4113f08b17e5bd11/src/Platforms/SQLite/Comparator.php#L14

I'm not sure if I should skip the test or keep it as is, since it might be interesting to assert that binary is indeed ignored.

Reply:
Is it really related to the binary collation? As I understand the test, it tries to change the properties of a binary column and assert that the diff corresponds to the changes. On SQLite, all columns are just `BINARY`, so the changes don't yield any diff.

If that's the case, we should just skip the test. We don't _want_ the comparison to behave this way on SQLite, it's _against_ our expectations in the base class but we consider it acceptable.